Softeq.XToolkit.Bindings.Droid/Bindable/BindableRecyclerViewAdapter.cs

```diff
@@ -335,6 +337,9 @@ public class BindableRecyclerViewAdapter<TKey, TItem, TItemHolder> : BindableRec
                 _subscription = new NotifyCollectionKeyGroupChangedEventSubscription<TKey, TItem>(dataSourceNew, NotifyCollectionChanged);
             }
 
+            HeaderSectionViewHolder = headerSectionViewHolder;
+            FooterSectionViewHolder = footerSectionViewHolder;
```

Comment:
Also, it seems like the issue is valid for all the rest of `BindableRecyclerViewAdapter` implementations, it's probably better to fix them all together.

Looks like we need to make these properties in the base class
https://github.com/Softeq/XToolkit.WhiteLabel/blob/45ecf6a535decc17d4d0b3bb7e211aa2a1dbb475/Softeq.XToolkit.Bindings.Droid/Bindable/BindableRecyclerViewAdapter.cs#L38-L40
also read-only

Reply:
Fixed